### PR TITLE
terraform 0.5.1

### PR DESCRIPTION
**Pull request by number5**
new version

### CODE REVIEW

**Comment by number5** [comment]
hmm, have no idea why yosemite build failed

File: Library/Formula/terraform.rb [hunk 1]
```diff
@@ -3,50 +3,54 @@
...
 github.com/mitchellh/gox e8e6fd4fe12510cc46893dff18c5188a6a6dc549
 github.com/mitchellh/iochan b584a329b193e206025682ae6c10cdbe03b0cd77
+github.com/mitchellh/go-linereader 07bab5fdd9580500aea6ada0e09df4aa28e68abd
 github.com/mitchellh/mapstructure 442e588f213303bec7936deba67901f8fc8f18b1
 github.com/mitchellh/osext 0dd3f918b21bec95ace9dc86c7e70266cfc5c702
-github.com/mitchellh/packer c8b3dfff5fa8470f7d639c3e6b09f15ff09fa7c2
+github.com/mitchellh/packer 350a5f8cad6a0e4c2b24c3049a84c4f294416e16
 github.com/mitchellh/panicwrap 45cbfd3bae250c7676c077fb275be1a2968e066a
 github.com/mitchellh/prefixedio 89d9b535996bf0a185f85b59578f2e245f9e1724
 github.com/mitchellh/reflectwalk 242be0c275dedfba00a616563e6db75ab8f279ec
```
Comment (xu-cheng) [inline Library/Formula/terraform.rb]: Can you indent this please? It looks weird.

**Comment by number5** [comment]
@xu-cheng done

File: Library/Formula/terraform.rb [hunk 2]
```diff
@@ -3,64 +3,68 @@
...
     cellar :any
-    sha256 "ebf80d6dadaac89c0492030a44ae6ae3c386754fe467cf705c01a0334cded4ef" => :yosemite
-    sha256 "5816bc8ad31ee155e9783b948f297bca86096189f65c13b3e66ee3c138d29e45" => :mavericks
-    sha256 "7c2ef7e1bb3ae1da851d05d0d67c7d86f155b0fc4f6b8b03af844063e8c891d3" => :mountain_lion
+  end
+
+  # patch from upstream, will be included in next release
+  # fix regression in networkacl with conflicts with
+  patch do
+    url "https://github.com/hashicorp/terraform/commit/af09f257c8d5522ce78f9a9088d04f653fb73ec9.patch"
```
Comment (mikemcquaid) [inline Library/Formula/terraform.rb]: Use `.diff` here please. Also, you can shorten the commit hash to `af09f2.diff`

**Comment by number5** [comment]
@mikemcquaid done

**Comment by number5** [comment]
ping

**Comment by number5** [comment]
@mikemcquaid @xu-cheng do I still need to do anything?

File: Library/Formula/terraform.rb (hunk 2 above)
Comment (mistydemeo) [inline Library/Formula/terraform.rb]: For reference Github isn't appending git version numbers to `.patch` files anymore - these should be safe to use now.

**Comment by mikemcquaid** [comment]
@number5 Nope, sorry for the delay!

Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by number5** [comment]
@mikemcquaid it's alright :smiley: thanks for the great works

**Comment by dcarley** [comment]
Second that. Thanks @number5 for keeping this updated.

**Pull request closed without merging**